tensorflow_addons/layers/tests/spectral_normalization_test.py

```diff
@@ -0,0 +1,168 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+
+import numpy as np
+import pytest
+import tensorflow as tf
+
+from tensorflow_addons.layers import spectral_normalization
+from tensorflow_addons.utils import test_utils
+
+
+def test_keras():
+    input_data = np.random.random((10, 3, 4)).astype(np.float32)
+    test_utils.layer_test(
+        spectral_normalization.SpectralNormalization,
+        kwargs={"layer": tf.keras.layers.Dense(2), "input_shape": (3, 4)},
+        input_data=input_data,
+    )
+
+
+def test_from_to_config():
+    base_layer = tf.keras.layers.Dense(1)
+    sn = spectral_normalization.SpectralNormalization(base_layer)
+    config = sn.get_config()
+
+    new_sn = spectral_normalization.SpectralNormalization.from_config(config)
+    assert sn.power_iterations == new_sn.power_iterations
+
+
+def test_save_load_model(tmpdir):
+    base_layer = tf.keras.layers.Dense(1)
+    input_shape = [1]
+
+    inputs = tf.keras.layers.Input(shape=input_shape)
+    sn_layer = spectral_normalization.SpectralNormalization(base_layer)
+    model = tf.keras.models.Sequential(layers=[inputs, sn_layer])
+
+    # initialize model
+    model.predict(np.random.uniform(size=(2, 1)))
+
+    model.save(tmpdir / "test.h5")
+    new_model = tf.keras.models.load_model(tmpdir / "test.h5")
```

Comment:
Hi @charlielito , sorry for the misleading info. Seems that keras does not accept `LocalPath` object. Let's try to wrap `tmpdir / "test.h5"` with `str`, that is, use `str(tmpdir / "test.h5")`.

Reply:
Strangely, when I run it locally it works